livesync-library.js

```diff
@@ -329,7 +329,7 @@ module.exports = (function () {
 
         _handleData(socketId, data) {
             const reportType = data.readUInt8(),
-                infoBuffer = data.slice(Buffer.byteLength(reportType), data.length);
+                infoBuffer = data.slice(Buffer.byteLength(reportType.toString()), data.length);
```

Comment:
Better add REPORT_LENGTH constant equal to 1 and use it here. Depending on the default encoding Buffer.byteLength() might return more than 1 for a char.